src/Parallel/centrality/betweenness.jl

```diff
@@ -1,16 +1,56 @@
-betweenness_centrality(g::AbstractGraph, vs::AbstractVector=vertices(g), distmx::AbstractMatrix=weights(g); normalize=true, endpoints=false, parallel=:distributed) = 
-parallel == :distributed ? distr_betweenness_centrality(g, vs, distmx; normalize=normalize, endpoints=endpoints) :
-threaded_betweenness_centrality(g, vs, distmx; normalize=normalize, endpoints=endpoints)
+betweenness_centrality(
+    g::AbstractGraph,
+    vs::AbstractVector = vertices(g),
+    distmx::AbstractMatrix = weights(g);
+    normalize = true,
+    endpoints = false,
+    parallel = :distributed,
+) = parallel == :distributed ?
+    distr_betweenness_centrality(
```

Comment:
this should be indented 4 spaces since it's a continuation of the statement on the previous line.